Assets/HoloToolkit/UX/Resources/Models/Cursor/Cursor_move/CursorRing.fbm.meta

```diff
@@ -0,0 +1,10 @@
+fileFormatVersion: 2
+guid: 77822bd0488fb7a4fb65cdeba7633923
```

Comment:
We should probably disable the material import for `CursorRing.fbx`

Reply:
We should disable material import for all the Cursor_Scale meshes as well.

Reply:
All the materials in the models folder should be moved over to the Materials folder.